samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
this is already supported by UITextField, so i don't have to add anything? 

---
In reply to: [105040618](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105040618) [](ancestors = 105040618,105038372,104793428)